r_py_cheatsheet.md

```diff
@@ -55,32 +55,39 @@ Working example (asap):
 
 ### R
 
-Purpose:
-Official documentation:
-Good help, tutorials:
+Purpose: Use filter find rows/cases where conditions are true.
+Official documentation: https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/filter
+Good help, tutorials: 
 
-1. ...
-2. ...
-3. ...
+1. [R for Data Science](https://r4ds.had.co.nz/transform.html#filter-rows-with-filter)
+2. https://www.guru99.com/r-select-filter-arrange.html#2
 
 Working example (asap):
 
 ```R
+library(tidyverse)
+library(ggplot2)
+
+# filter for cars with city miles per gallon less than 20
+greenCars <- filter(mpg, cty < 20)
+print(greenCars)
 ```
 
 ### Python
 
-Purpose:
-Documentation:
+Purpose:  Use filter find rows/cases where certain conditions are meet.
```

Comment:
```suggestion
Purpose:  Use "filter" to find rows/cases meeting certain conditions.<br>
```

Reply:
text is nice but you do not need that <br> I think.